apcd-cms/src/client/src/components/Extensions/EditExtensionModal/EditExtensionModal.tsx

```diff
@@ -0,0 +1,380 @@
+                              {opt.value}
+                            </option>
+                          )
+                      )}
+                    </Field>
+                    <ErrorMessage
+                      name="ext_status"
+                      component="div"
+                      className="text-danger"
+                    />
+                  </FormGroup>
+                </Col>
+                <Col md={6}>
+                  <FormGroup>
+                    <Label for="ext_outcome" className={styles.customLabel}>
+                      <strong>Extension Outcome</strong>
+                    </Label>
+                    <Field
+                      as="select"
+                      name="ext_outcome"
+                      id="ext_outcome"
+                      onChange={formik.handleChange}
+                      onBlur={formik.handleBlur}
+                      className={`form-control ${styles.viewRecord}`}
+                      value={formik.values.ext_outcome}
+                    >
+                      {outcomeOptions?.map((opt) => (
+                        <option
+                          className="dropdown-text"
+                          key={opt.key}
+                          value={opt.value}
+                        >
+                          {opt.value}
+                        </option>
+                      ))}
+                    </Field>
+                    <ErrorMessage
+                      name="ext_outcome"
+                      component="div"
+                      className="text-danger"
+                    />
+                  </FormGroup>
+                </Col>
+                <Col md={6}>
+                  <FormGroup>
+                    <Label for="notes" className={styles.customLabel}>
+                      <strong>Notes</strong>
+                    </Label>
+                    <Field
+                      as="textarea"
+                      name="notes"
+                      id="notes"
+                      rows="5"
+                      maxLength="2000" // Set the maxLength attribute
+                      onChange={formik.handleChange}
+                      onBlur={formik.handleBlur}
+                      className={`form-control ${styles.viewRecord}`}
+                    />
+                    <small
+                      className="form-text text-muted"
+                      style={{ fontStyle: 'italic' }}
+                    >
+                      2000 character limit
+                    </small>
+                    <ErrorMessage
+                      name="notes"
+                      component="div"
+                      className="text-danger"
+                    />
+                  </FormGroup>
+                </Col>
+              </Row>
+              <br />
+              <Alert color="danger" isOpen={showErrorMessage}>
+                Error: {errorMessage}
+              </Alert>
+              <Button
```

Comment:
```suggestion
                    <Button
                      type="primary"
                      attr="submit"
                      isLoading={isSubmitting}
                      onClick={() => setIsSuccess(false)}
                    >
                      Submit
                    </Button>
```
From suggestion above about using Core-component button rather than reactstrap